README.md

```diff
@@ -103,6 +104,40 @@ bash tear_down_environment.sh
     ```
 </p>
 
+## Local Debugging
+
+Firstly, run 'Create Local Db' script
+
+### On a Windows box
+```
+& '.\Create Local Db.ps1'
+```
+
+### On Linux/Mac box
+```
+bash create_local_db.sh
+```
+
+Secondly, copy and paste the connection string into your User Secrets file (in Visual Studio: src -> Right click on NHSD.BuyingCatalogue.API -> Manage User Secrets ) 
```

Comment:
Not everyone is using visual studio. I wonder whether we can use a link instead to refer the user to the official document regarding user secrets. It has platform specific instructions on there as well.

https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.1&tabs=windows

If we do this, then we can move this text under then "Local Debugging" heading as a summary.

Reply:
Updated the README.md according to our conversation